guides/tutorials/codelabs/safe-deployments/index.md

```diff
@@ -0,0 +1,287 @@
+---
+layout: single
+title:  "Safe Deployments"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+In this codelab you will improve a deployment process by adding safeguards to your deployments.
+
+This codelab is a continuation of the Kubernetes: code to prod codelab. It assumes you have an application set up and ready to deploy.
+
+# Setup
```

Comment:
The page title is at H1. I and others have been using H2 for main sections throughout the docs to differentiate.